test/functional/bundleadd/fix-missing-file/test.bats

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bats
+
+load "../../swupdlib"
+
+t1_hash="e6d85023c5e619eb43d5cfbfdbdec784afef5a82ffa54e8c93bda3e0883360a3"
+
+setup() {
+  clean_test_dir
+  tar -C "$DIR/web-dir/10" -cf "$DIR/web-dir/10/Manifest.MoM.tar" Manifest.MoM Manifest.MoM.signed
+  tar -C "$DIR/web-dir/10" -cf "$DIR/web-dir/10/Manifest.os-core.tar" Manifest.os-core Manifest.os-core.signed
+  tar -C "$DIR/web-dir/10" -cf "$DIR/web-dir/10/Manifest.test-bundle.tar" Manifest.test-bundle Manifest.test-bundle.signed
+  sudo chown root:root "$DIR/web-dir/10/files/$t1_hash"
+  tar -C "$DIR/web-dir/10/files" -cf "$DIR/web-dir/10/files/$t1_hash.tar" $t1_hash --exclude=$t1_hash/*
```

Comment:
Since t1_hash is just a normal file, the --exclude is not needed.

Without looking at the other tests, do we always pass --exclude in this case?